style.css

```diff
@@ -0,0 +1,6 @@
+* {
+  font-family: 'Oswald', sans-serif;
+}
+h1 {
+  font-family: 'Pacifico', cursive;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/New_html_proj/pullRequest?prid=5459572)